floodviz/utils.py

```diff
@@ -0,0 +1,54 @@
+import requests
+
+
+def parse_rdb(endpoint, params):
+    """
+        Takes a web service endpoint, a dictionary of key/value pairs for the query parameters
+        and returns all parsed data from the webservice.
+
+        ARGS:
+            endpoint - String specifying the web service url
+            params - dictionary holding url parameters
+                * keys for params should be the value you variable name for the query parameter
+                  you want to specify. Values for these keys should be the query parameter value
+                  you wish to use.
+
+                  i.e {'site_no' : '056777781'} -> '&site_no=056777781'
+
+        RETURNS:
+            url - url to request for data
+    """
+    all_data = []
+    try:
+        r = requests.get(endpoint, params)
+    except requests.exceptions.RequestException as e:
+        print(e)
+        print('- bad webservice url -')
+    else:
+        if r.status_code is 200:
+            content = iter(r.text.splitlines())
+            headers = None
+            for line in content:
+                # Skip commented lines
+                if line.startswith('#'):
```

Comment:
This statement will skip any line that begins with a # sign. After the beginning comments and the header lines, the rest of the lines are data and may begin with a # sign. This needs to be written so that only the initial lines with a # sign are skipped. Considering using a while loop to remove the initial comment lines and then proceed.